tardis/montecarlo/montecarlo_numba/formal_integral.py

```diff
@@ -743,12 +743,12 @@ def trapezoid_integration(array, h):
 
 
 @njit(**njit_dict_no_parallel)
-def intensity_black_body(nu, T):
+def intensity_black_body(nu, temperature):
     """Get the black body intensity at frequency nu
-    and temperature T"""
+    and temperature """
```

Comment:
Thank you for opening this PR!
Maybe reword this to something like- "Get the black body intensity.". The first sentence of the docstring should be in one line. 
And then adding a `Parameters` and `Returns` section in the docstring. Something like this- https://tardis-sn.github.io/tardis/contributing/development/code_quality.html#docstrings